Ingress/controllers/nginx-third-party/nginx/types.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nginx
+
+import (
+	"text/template"
+
+	"k8s.io/contrib/Ingress/controllers/nginx-third-party/ssl"
+
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+type nginxConfiguration struct {
+	BodySize                 string `json:"bodySize,omitempty" structs:",omitempty"`
```

Comment:
would be nice if you added a short comment above each one. Something like:
```
// Minimum is TLS 1.1 because 1.0 suffers a downgrade attack. 
sslProtocols

// Connection keepalive. Higher keepalive usually means fewer connections. 
// See https://www.nginx.com/blog/http-keepalives-and-web-performance/ for more details. 
keepAlive
```
1.0 is BEASTed right? or am i confusing them.

We add comments in the main repo (https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go) and convert it to documentation through swagger (http://kubernetes.io/third_party/swagger-ui/). I've found that adding comments usually forces one to thing about the field, but this is just a suggestion.